theme/ItaliaTheme/Blocks/_iconBlocks.scss

```diff
@@ -154,6 +167,7 @@
     .iconblock-icon {
       margin-bottom: 2rem;
       text-align: center;
+      min-height: 5rem;
```

Comment:
Io questo propongo di farlo in un altro modo. Ho provato ad applicare questa modifica ad un blocco già creato, e viene così. Da notare le icone allineate in alto:

<img width="1273" alt="image" src="https://user-images.githubusercontent.com/14176608/235156506-62eb3b7a-666f-4bd7-97b9-7817f248f875.png">

Potresti invece provare a non mettere questa min-height e a cambiare lo stile dell'svg interno così:

<img width="1873" alt="image" src="https://user-images.githubusercontent.com/14176608/235156675-0db7d3e9-c96d-49aa-9b28-a5275ae75eb5.png">

Cosa ne pensate? Taggo anche @andreabaglioni, che non so se abbia le notifiche di github attive, in caso @Wagner3UB pingalo tu

Reply:
@pnicolli d'accordo, meglio così

Reply:
Bene. Ora questa riga non serve più, intendo la min-height, quindi va rimossa

Reply:
Commit fatto e non pushato, scusa